docker/build.sh

```diff
@@ -1,5 +1,6 @@
 #! /usr/bin/env bash
```

Comment:
I wonder if we want to remove these (or at least just `push.sh`) if they're obsolete? Having a single way to do things seems sensible/less confusing.